_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/referrer.js

```diff
@@ -11,22 +11,21 @@
           referrerHTML.insertBefore('main');
           break;
       }
-      
-      $('#referral-alert .close').on("click", function() {
-        $('#referral-alert').addClass('hide');
+      $('#referral-alert .pf-c-alert__action').on("click", function() {
```

Comment:
Should this be `pf-c-alert__icon`?  per html documentation here 
 - https://www.patternfly.org/v4/documentation/core/components/alert

Can the custom event listeners for PatternFly components be put in their own js file with a Drupal behavior?